### PR TITLE
Update to 3.6.0

### PR DESCRIPTION
**Pull request by vtourraine**
- marked feed & suggest APIs as deprecated (#127)
- fixed compilation warnings (#125)

### CODE REVIEW

File: MendeleyKit.podspec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name = "MendeleyKit"
-  s.version = "3.5.2"
+  s.version = "3.5.3"
```
Comment (Aaron-Tunney) [inline MendeleyKit.podspec:3]: Deprecating APIs feels like more than a patch release.

Probably not v4.0.0 but probably v3.6.0?

Comment (vtourraine) [reply MendeleyKit.podspec:3]: I considered it, but at this stage it’s just a warning... so I’m not sure how SemVer handles that.

Comment (vtourraine) [reply MendeleyKit.podspec:3]: > Minor version Y (x.Y.z | x > 0) MUST be incremented if new, backwards compatible functionality is introduced to the public API. It MUST be incremented if any public API functionality is marked as deprecated.

Haaaa, ok, I’ll bump it to 3.6.0 then.

Comment (Piotrek605) [reply MendeleyKit.podspec]: I think I agree with Aaron, the last number is meant to be for bug fixes and the middle one for adding features, I guess deprecating API is a bit like removing features?

**Review by Piotrek605 — APPROVED** [review]

**Review by Aaron-Tunney — APPROVED** [review]

**Pull request merged**